### DISCUSSION TITLE
can 支持印刷体公式吗

### DISCUSSION THREAD

**Discussion by paul007008** [PaddlePaddle/PaddleOCR#12503] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by WYH150123** [comment]
可以支持的啊，手写是更复杂的情况，对于印刷体公式我有在训练，有什么问题，可以和我交流

**Comment by SWHL** [comment]
@WYH150123 你好，你这边有训过在印刷体公式识别上的效果吗？整体识别准确率咋样啊，最近也考虑用这个试试印刷体

**Comment by SWHL** [comment]
看到CAN repo下有小伙伴说，训练印刷体识别的效果不好。 [issue 26](https://github.com/LBH1024/CAN/issues/25)
